### ISSUE TITLE
Undeclared identifier 'guiIconsPtr' when using custom_icons.rgi.h from rGuiIcons

### ISSUE THREAD

**Issue by SinixND** [raysan5/rguiicons#3]
I might have found an issue with the use of a custom icon file exportet from rGuiIcons:

When I wanted to include a custom icon header in my files, I got the following error message:

`In file included from ./src/main.cpp:7:`
`/usr/include/raygui.h:4216:42: error: use of undeclared identifier 'guiIconsPtr'; did you mean 'guiIcons'?`
`unsigned int *GuiGetIcons(void) { return guiIconsPtr; }`

I solved it by adding 
`static unsigned int *guiIconsPtr = guiIcons;`
to the bottom of the custom_icons.rgi.h file.

I think it has to do with the way raygui detects the #220# (in my case) within a text and that it uses the `guiIconsPtr` to get the icon data.
I was not able to find the function where it retrieves the actual icon data (to check, whether the pointer hides in there somewhere).
Basically, this pointer gets deactivated within raygui.h when RAYGUI_CUSTOM_ICONS is defined.

Therefore I think the solution might be to either exclude the pointer from the `#ifdefined`, or re-add it in the exported `.h`
